tests.py

```diff
@@ -23,6 +23,36 @@ def test_add(self):
             self.assertEqual(self.results[i], temp)
 
 
+    def test_infinity(self):
+        P = self.results[0]
+        o = Point(None, None, self.curve)
```

Comment:
If we'd change this line to `o = Point.infinity()` we'd get a `PointsOnDifferentCurveError` below when adding `P` and `o`.